source/applications/basic/visualization/read_zdf_vis_3d.py

```diff
@@ -75,6 +75,8 @@ def _display_pointcloud(rgb, xyz):
         rgb: RGB image
         xyz: X, Y and Z images (point cloud co-ordinates)
 
+    Returns None
+
```

Comment:
Should be separate commit. I assume this is because CI fails? Did you rebase first? Is so, then this should be the first commit.

Reply:
I've split the commits and placed this one on the first place.